src/App.vue

```diff
@@ -120,6 +120,7 @@ const centsFractionDigits = ref(3);
 const decimalFractionDigits = ref(5);
 const showVirtualQwerty = ref(false);
 const midiOctaveOffset = ref(-1);
+const intervalMatrixIndexing = ref<"zero" | "one">("zero");
```

Comment:
This could be replaced with a number called `intervalMatrixIndexOffset` that gets added to the headers if that's the intended effect.

Reply:
In my previous iterations of attempting that, it seems the radio buttons don't like me to make its associated type a `number` instead of a `radio`:

Internal server error: Unnecessary value binding used alongside v-model. It will interfere with v-model's behavior.
  Plugin: vite:vue
  File: /home/inthar/Downloads/scale-workshop-main/src/views/AnalysisView.vue
  160|            <input
  161|              type="number"
  162|              id="indexing-zero"
     |                               ^
  163|              value="0"
  164|              v-model="intervalMatrixIndexing"
      at Object.createCompilerError (/home/inthar/Downloads/scale-workshop-main/node_modules/@vue/compiler-core/dist/compiler-core.cjs.js:19:19)
      at createDOMCompilerError (/home/inthar/Downloads/scale-workshop-main/node_modules/@vue/compiler-dom/dist/compiler-dom.cjs.js:2483:25)
      at checkDuplicatedValue (/home/inthar/Downloads/scale-workshop-main/node_modules/@vue/compiler-dom/dist/compiler-dom.cjs.js:2547:29)
      at transformModel (/home/inthar/Downloads/scale-workshop-main/node_modules/@vue/compiler-dom/dist/compiler-dom.cjs.js:2579:29)
      at buildProps (/home/inthar/Downloads/scale-workshop-main/node_modules/@vue/compiler-core/dist/compiler-core.cjs.js:4777:48)
      at Array.postTransformElement (/home/inthar/Downloads/scale-workshop-main/node_modules/@vue/compiler-core/dist/compiler-core.cjs.js:4374:38)
      at traverseNode (/home/inthar/Downloads/scale-workshop-main/node_modules/@vue/compiler-core/dist/compiler-core.cjs.js:2225:19)
      at traverseChildren (/home/inthar/Downloads/scale-workshop-main/node_modules/@vue/compiler-core/dist/compiler-core.cjs.js:2167:9)
      at traverseNode (/home/inthar/Downloads/scale-workshop-main/node_modules/@vue/compiler-core/dist/compiler-core.cjs.js:2218:13)
      at traverseChildren (/home/inthar/Downloads/scale-workshop-main/node_modules/@vue/compiler-core/dist/compiler-core.cjs.js:2167:9)

Reply:
In my previous iterations of attempting that, it seems the radio buttons don't want me to make their associated type a `number` instead of a `radio`:

Internal server error: Unnecessary value binding used alongside v-model. It will interfere with v-model's behavior.
  Plugin: vite:vue
  File: /home/inthar/Downloads/scale-workshop-main/src/views/AnalysisView.vue
  160|            <input
  161|              type="number"
  162|              id="indexing-zero"
     |                               ^
  163|              value="0"
  164|              v-model="intervalMatrixIndexing"
      at Object.createCompilerError (/home/inthar/Downloads/scale-workshop-main/node_modules/@vue/compiler-core/dist/compiler-core.cjs.js:19:19)
      at createDOMCompilerError (/home/inthar/Downloads/scale-workshop-main/node_modules/@vue/compiler-dom/dist/compiler-dom.cjs.js:2483:25)
      at checkDuplicatedValue (/home/inthar/Downloads/scale-workshop-main/node_modules/@vue/compiler-dom/dist/compiler-dom.cjs.js:2547:29)
      at transformModel (/home/inthar/Downloads/scale-workshop-main/node_modules/@vue/compiler-dom/dist/compiler-dom.cjs.js:2579:29)
      at buildProps (/home/inthar/Downloads/scale-workshop-main/node_modules/@vue/compiler-core/dist/compiler-core.cjs.js:4777:48)
      at Array.postTransformElement (/home/inthar/Downloads/scale-workshop-main/node_modules/@vue/compiler-core/dist/compiler-core.cjs.js:4374:38)
      at traverseNode (/home/inthar/Downloads/scale-workshop-main/node_modules/@vue/compiler-core/dist/compiler-core.cjs.js:2225:19)
      at traverseChildren (/home/inthar/Downloads/scale-workshop-main/node_modules/@vue/compiler-core/dist/compiler-core.cjs.js:2167:9)
      at traverseNode (/home/inthar/Downloads/scale-workshop-main/node_modules/@vue/compiler-core/dist/compiler-core.cjs.js:2218:13)
      at traverseChildren (/home/inthar/Downloads/scale-workshop-main/node_modules/@vue/compiler-core/dist/compiler-core.cjs.js:2167:9)

Reply:
It's one of these things that seem like they ought to work but don't.

Reply:
Yeah, you need an auxillary ref and a watcher if you want to use `v-model` it's better to just emit the correct values from the radio buttons directly.

Reply:
What can I use to label the data instead of a `v-model`?